manifests/manifest_celery_worker_dev.yml

```diff
@@ -1,7 +1,20 @@
 ---
-inherit: manifest_dev.yml
+path: ../
+memory: 1G
+stack: cflinuxfs2
+buildpack: python_buildpack
 env:
+  CACHE_ALL_REQUESTS: True
   NEW_RELIC_APP_NAME: fec | celery worker | dev
+  NEW_RELIC_CONFIG_FILE: /home/vcap/app/newrelic.ini
+  NEW_RELIC_ENV: development
+  NEW_RELIC_LOG: stdout
+  WEB_CONCURRENCY: 4
+services:
+  - fec-api-search
+  - fec-redis
+  - fec-creds-dev
+  - fec-s3-dev
 applications:
   - name: celery-worker
     instances: 2
```

Comment:
See note in the `prod` manifest file for this, it seems off to me.

Reply:
I caught up with @LindsayYoung and confirmed my suspicions, we want just one instance of this because it is the piece that handles the scheduled tasks.  With multiple instances we don't want to run the risk of tasks firing off more than once!

If you wouldn't mind adjusting this back to `1`, @lbeaufort, then this PR will be ready to go. :-)

Reply:
Okay, total FAIL on my part, I'm sorry!  I mixed this manifest up with the `celery-beat` manifest above.  We're actually okay here, and it may make sense to keep 2 instances just so that we continue being able to test the setup of multiple instances, especially if we make any changes to the setup of Celery.